app/main.py

```diff
@@ -1,2 +1,65 @@
 class Dictionary:
-    pass
+    def __init__(self):
+        self.capacity = 8
+        self.length = 0
+        self.load = int(self.capacity * 2 / 3)
+        self.tab = [[] for _ in range(self.capacity)]
```

Comment:
```suggestion
        self.table = [[] for _ in range(self.capacity)]
```
Use more descriptive names